frontend/app/views/fragments/offer/image.scala.html

```diff
@@ -0,0 +1,9 @@
+@(imgSrc: String)
+
+@import views.support.Asset
+
+<div class="promo-letterbox">
```

Comment:
I think this should be combined with `offer/callout` into a generic letterbox component which takes an Option of a title and an option of a tone class